rsconnect/bundle.py

```diff
@@ -760,6 +765,118 @@ def make_html_bundle(
     return bundle_file
 
 
+def pack_extra_files(
```

Comment:
This function is very similar to make_html_bundle_content. Can we factor out the large common section near the end of the function?